assembly/integer/u256.ts

```diff
@@ -100,6 +106,28 @@ export class u256 {
     );
   }
 
+  static fromUint8ArrayLE(array: Uint8Array): u256 {
+    assert(array.length && (array.length & 31) == 0);
+    var buffer = <ArrayBuffer>array.buffer;
```

Comment:
Casting to `ArrayBuffer` is unnecessary for typedarray's buffer. Plz skip this

Reply:
Fixed